tests/Core/RouterTest.php

```diff
@@ -24,6 +24,7 @@ public function test_catch_exception_when_visiting_wrong_url()
         ];
         $this->router = new Router($routes, $this->request);
         $this->get('/wrong-url');
+    
```

Comment:
1. Why are you touching `RouterTest` within this issue?
2. I am not sure this new line is actually needed, see [my answer](https://github.com/MagedAhmad/TrendingRepos/pull/23/commits/088fcb5ccbb26d4c04fa97786cd7676a2f901dbf#diff-095ca4a5b5483dfff6e638f8dbdb07ccR61) to your comment in the previous PR.

Reply:
I touched this file when i got merge conflicts, i didn't want to touch any Issue-21 related files believe me :laughing: